### PR TITLE
Prisma cloud compute authentication doc

### PR DESCRIPTION
**Pull request by GuyAfik**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
add documentation section regarding user roles in Prisma Cloud Compute

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/README.md [hunk 1]
```diff
@@ -35,6 +36,23 @@ To send alerts from Prisma Cloud Compute to Cortex XSOAR, you need to create an
 4. Click **Test** to validate the integration.
 5. Click **Done** to save the integration.
 
+## Configure Prisma Cloud Compute User Roles 
+
+* In order to access Prisma Cloud Compute resources, a user must be assigned with a role.
+* Without insufficient user roles, commands/fetching incidents might not work.
```
Comment (Itay4) [inline Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/README.md:42]:
```suggestion
* Without sufficient user roles, commands/fetching incidents might not work.
```

**Review by Itay4 — COMMENTED** [review]
no need for the required permissions in the command description

just add the following:
1. all required roles at the readme file (can be below the section you added on how to add roles)
2. for each command, add `#### Requires Role`

File: Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute_description.md [hunk 2]
```diff
@@ -10,3 +10,6 @@ This integration provides the ability to import **Palo Alto Networks - Prisma Cl
 4. On the left, select **Demisto** from the provider list.
 5. On the right, select the alert triggers. Alert triggers specify which alerts are sent to Cortex XSOAR.
 6. Click **Save** to save the alert profile.
+7. Make sure you configure the user role to be at least `auditor`, otherwise you will not be able to fetch the alerts.
+
+[View Integration Documentation](https://xsoar.pan.dev/docs/reference/integrations/palo-alto-networks-prisma-cloud-compute)
```
[the remark this answers is not in the document]
Comment (GuyAfik) [reply Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute_description.md:15]: done

File: Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/README.md (hunk 1 above)
Comment (GuyAfik) [reply Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/README.md:42]: done

**Comment by GuyAfik** [comment]
> no need for the required permissions in the command description
> 
> just add the following:
> 
> 1. all required roles at the readme file (can be below the section you added on how to add roles)
> 2. for each command, add `#### Requires Role`

1. not sure what you mean, right now there is screenshot with a description of all the available user roles and their descriptions
2. done

**Comment by GuyAfik** [comment]
@Itay4 Feel free to review, secrets issue is a known issue  - once the PR will be approved, @abaumgarten would run the secrets check locally

File: Packs/PrismaCloudCompute/ReleaseNotes/1_2_1.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Palo Alto Networks - Prisma Cloud Compute
+- General Documentation Improvements.
```
Comment (Itay4) [inline Packs/PrismaCloudCompute/ReleaseNotes/1_2_1.md:4]:
```suggestion
- Documentation and metadata improvements.
```

**Review by Itay4 — APPROVED** [review]

File: Packs/PrismaCloudCompute/ReleaseNotes/1_2_1.md (hunk 3 above)
Comment (GuyAfik) [reply Packs/PrismaCloudCompute/ReleaseNotes/1_2_1.md:4]: will  update now

**Pull request merged**